python/pyarrow/tests/conftest.py

```diff
@@ -311,3 +320,23 @@ def s3_server(s3_connection):
         finally:
             if proc is not None:
                 proc.kill()
+
+
+@pytest.fixture(scope='session')
+def gcs_server():
+    port = find_free_port()
+    env = os.environ.copy()
+    args = [sys.executable, '-m', 'testbench', '--port', str(port)]
+    proc = None
+    try:
+        proc = subprocess.Popen(args, env=env)
+    except OSError:
+        pytest.skip('`gcs test bench` command cannot be located')
```

Comment:
Why this message? Is "gcs test bench" an actual command?

Reply:
most copy and past from s3, I rephrased slightly.